From 5749e02400bd5947812837b8e325431da35bd5e5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Krause?= <joerg.krause@embedded.rocks>
Date: Wed, 22 Feb 2017 20:45:29 +0100
Subject: [PATCH] libunwind: do not link libc statically for ARM musl
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jörg Krause <joerg.krause@embedded.rocks>
---
 src/libunwind/build.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libunwind/build.rs b/src/libunwind/build.rs
index ea0d769783..715c04eac7 100644
--- a/src/libunwind/build.rs
+++ b/src/libunwind/build.rs
@@ -14,7 +14,8 @@ fn main() {
     let target = env::var("TARGET").expect("TARGET was not set");
 
     if target.contains("linux") {
-        if target.contains("musl") && !target.contains("mips") {
+        if target.contains("musl") && !target.contains("mips")
+                                   && !target.contains("arm") {
             println!("cargo:rustc-link-lib=static=unwind");
         } else if !target.contains("android") {
             println!("cargo:rustc-link-lib=gcc_s");
-- 
2.11.1

